src/raftstore/store/config.rs

```diff
@@ -146,7 +147,8 @@ impl Default for Config {
             messages_per_tick: 4096,
             max_peer_down_duration: ReadableDuration::minutes(5),
             max_leader_missing_duration: ReadableDuration::hours(2),
-            abnormal_leader_missing_duration: ReadableDuration::minutes(2),
+            abnormal_leader_missing_duration: ReadableDuration::minutes(10),
+            peer_stale_state_check_interval: ReadableDuration::minutes(5),
```

Comment:
5 m seems a little long.

Reply:
Any suggestions? Do 2 minutes fit?

Reply:
I think 5 is OK. As there is an alert rule already.